src/main/windows/windowManager.js

```diff
@@ -178,12 +178,57 @@ export function flashFrame(flash) {
   }
 }
 
-export function setOverlayIcon(overlayDataURL, description) {
+function drawBadge(text, small) {
+  const scale = 2; // should rely display dpi
```

Comment:
0/5 on if we do this, but I wonder if it's safe and secure to instead have a template for this HTML and CSS so that it's not all in JavaScript, and then we render that template when we need to.

Reply:
should we add that as a HW ticket? or do you prefer to fix it in this PR?

Reply:
It's not required right now, would be 100% on board with making a HW ticket for that. I'll do that now.